tcs34725.py

```diff
@@ -123,6 +123,23 @@ def _temperature_and_lux(self, data):
         cct = 449.0 * n**3 + 3525.0 * n**2 + 6823.3 * n + 5520.33
         return cct, y
 
+    def html_rgb(self):
+        r, g, b, c = self.read(raw=True)
+        red = int((r/c) * 256)
+        green = int((g/c) * 256)
+        blue = int((b/c) * 256)
+
+        gammatable = {}
+        for x, y in enumerate(range(0, 256)):
+            x = x / 255
+            x = pow(x, 2.5)
+            x = x * 255
+            gammatable[y] = x
```

Comment:
It would be much better to compute this on the fly for the three values you are going to use it for, instead of allocating a big chunk of memory and computing this for all the 256 possible values.